pkg/utils/spinner.go

```diff
@@ -23,3 +27,18 @@ func NewSpinner(message string, maxRetries int, function func() error) error {
 
 	return nil
 }
+
+func IsRunningInForegroundOrDefault() bool {
+	// Get the foreground process group ID of the terminal
+	foregroundPGID, err := unix.IoctlGetInt(int(os.Stdout.Fd()), unix.TIOCGPGRP)
```

Comment:
I've never really asked, but what platforms does the skupper CLI run on? The CLI strikes me as something that should maybe be able to run on windows. It at least compiles on main now. I assume this would not compile.

Reply:
@c-kruse After creating this pr, I've seen that the CI didn't pass the windows compilation, so no, this solution is not valid.

Reply:
@nluaces  :+1: As a side note, I don't think it's totally unreasonable to fix this for only unix-y systems. I don't even know if windows has the same concept of foreground/background.

My suggestion would be to have a spinner_windows.go implementation with IsRunningInForegroundOrDefault that is hardcoded true, then a spinner_unix.go with appropriate build tags with this solution.

Reply:
thanks @c-kruse !